### ISSUE TITLE
Transfer Learning with SWINIR model

### ISSUE THREAD

**Issue by VickkyMama** [JingyunLiang/SwinIR#143]
Hi,

I'm trying to finetune the swinir pretrained model using my own dataset for superresolution. Pls anyone knows how to go about it?

**Comment by XiangFeng66** [comment]
@VickkyMama Hello, when I used my own data set to fine-tune Swinir on the classic SR task, I found that the L1 loss did not decrease.  Can we have a chat about fine-tuning?
